### PR TITLE
Added Dune Legacy

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/dunelegacy/discussion/263161/thread/06051a04ed/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://sourceforge.net/p/dunelegacy/patches/8/ https://sourceforge.net/p/dunelegacy/patches/10/

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.sourceforge.DuneLegacy

File: dunelegacy.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>dunelegacy.desktop</id>
```
Comment (hfiguiere) [inline dunelegacy.appdata.xml:3]: you put the proper app-id here.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: dunelegacy.appdata.xml (hunk 1 above)
Comment (Mailaender) [reply dunelegacy.appdata.xml:3]: It gets renamed to @flathub style.

Comment (hfiguiere) [reply dunelegacy.appdata.xml:3]: yes but still. this app-id is completely invalid.

Comment (Mailaender) [reply dunelegacy.appdata.xml:3]: What is the proper app-id here?

Comment (hfiguiere) [reply dunelegacy.appdata.xml:3]: according the to manifest **YOU** wrote: `net.sourceforge.DuneLegacy`

Comment (hfiguiere) [reply dunelegacy.appdata.xml:3]: Note: this doesn't block.

File: net.sourceforge.DuneLegacy.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+modules:
+  - name: dunelegacy
+    buildsystem: autotools
+    post-install:
+      - install -Dm 0644 dunelegacy.desktop ${FLATPAK_DEST}/share/applications/dunelegacy.desktop
+      - install -Dm 0644 dunelegacy.appdata.xml ${FLATPAK_DEST}/share/appdata/dunelegacy.appdata.xml
+      - install -Dm 0644 dunelegacy-128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/dunelegacy.png
+    sources:
+      - type: git
+        url: https://git.code.sf.net/p/dunelegacy/code
```
Comment (barthalion) [inline net.sourceforge.DuneLegacy.yaml:26]: Please configure https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

File: dunelegacy.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline dunelegacy.appdata.xml:1]: This file should be submitted upstream.

File: dunelegacy.appdata.xml (hunk 1 above)
Comment (barthalion) [reply dunelegacy.appdata.xml:3]: I concur though; there's no point in doing it wrong in the downstream repo.

Comment (Mailaender) [reply dunelegacy.appdata.xml:3]: My reading of the AppStream metadata is that this _can_ be a reverse domain. Note that this file was created before that was even suggested in the standard.

File: dunelegacy.appdata.xml (hunk 3 above)
Comment (Mailaender) [reply dunelegacy.appdata.xml:1]: https://sourceforge.net/p/dunelegacy/patches/8/ done, but I don't see interest in upstream picking it up.

File: net.sourceforge.DuneLegacy.yaml (hunk 2 above)
Comment (Mailaender) [reply net.sourceforge.DuneLegacy.yaml:26]: Added

File: dunelegacy.appdata.xml (hunk 1 above)
Comment (hfiguiere) [reply dunelegacy.appdata.xml:3]: so if it was created before, that mean it is upstream, doesn't it? In that case why is it here?

Comment (Mailaender) [reply dunelegacy.appdata.xml:3]: Upstream ignored my patch.

**Comment by Mailaender** [comment]
bot, build net.sourceforge.DuneLegacy

**Pull request closed without merging**